ApiDoctor.Console/Program.cs

```diff
@@ -231,7 +231,7 @@ private static async Task RunInvokedMethodAsync(CommandLineOptions origCommandLi
             if (returnSuccess)
             {
                 if (issues.Errors.Any() ||
-                    (issues.Warnings.Any() && !options.IgnoreErrors))
+                    (issues.Warnings.Any() && !options.IgnoreErrors && !options.IgnoreWarnings))
```

Comment:
@finsharp I think this would make more sense

```
if ( (issues.Errors.Any() && !options.IgnoreErrors) || 
            (issues.Warnings.Any() && !options.IgnoreWarnings) )
{
returnSuccess = false;
}
```

As it is, if there are any errors, `returnSuccess = false` even if `options.IgnoreErrors = true`

Reply:
I think thats better. Fixed.